sections/main-product.liquid

```diff
@@ -255,7 +256,7 @@
                     {% render 'icon-plus' %}
                   </button>
                 </quantity-input>
-                <div class="quantity__rules caption">
+                <div class="quantity__rules caption no-js-hidden">
```

Comment:
**Not a blocker. A11Y**
This reads as `Increments of 7 **BULLET** Minimum of 49 **BULLET** Maximum of 1183`, which IMO, isn't great.

We may be able to use empty alternative text on the `content `property by adding a `/` and setting its content to empty:
`  content: '\2022' \ '';`

Can also be addressed in a separate PR.

Reply:
Issue created - https://github.com/Shopify/dawn/issues/2275